pages/style-tokens/color/overview.md

```diff
@@ -21,13 +21,13 @@ subnav:
 ---
 
 ## Introduction
-USWDS organizes its colors tokens into [project tokens]({{ site.baseurl }}/style-tokens/color/theme-tokens/) and [system tokens]({{ site.baseurl }}/style-tokens/color/system-tokens/). System tokens are the complete set of colors available to the design system. Project tokens are a smaller, role-based subset customized to your project's individual identity, tone, and needs. It will use only a few color families from the broader set of 23 color families available in the system token set.
+USWDS organizes its colors tokens into [theme tokens]({{ site.baseurl }}/style-tokens/color/theme-tokens/), [state tokens]({{ site.baseurl }}/style-tokens/color/theme-tokens/), and [system tokens]({{ site.baseurl }}/style-tokens/color/system-tokens/). System tokens are the complete set of colors available to the design system. Project tokens are a smaller, role-based subset customized to your project's individual identity, tone, and needs. It will use only a few color families from the broader set of 23 color families available in the system token set.
```

Comment:
Some copyedits for clarity

Reply:
It looks like the system tokens link needs to be updated. It's pointing to theme-tokens.

Reply:
```suggestion
USWDS organizes its colors tokens into [theme tokens]({{ site.baseurl }}/style-tokens/color/theme-tokens/), [state tokens]({{ site.baseurl }}/style-tokens/color/state-tokens/), and [system tokens]({{ site.baseurl }}/style-tokens/color/system-tokens/). System tokens are the complete set of colors available to the design system. Project tokens are a smaller, role-based subset customized to your project's individual identity, tone, and needs. It will use only a few color families from the broader set of 23 color families available in the system token set.
```

Reply:
Do you mean `state tokens`? I updated that link

Reply:
Inline code fix suggestions/commits is very cool...

Reply:
Yep!